pkg/api/aim/query/query.go

```diff
@@ -217,6 +220,22 @@ func (pq *parsedQuery) parseAttribute(node *ast.Attribute) (any, error) {
 		}
 		attribute := string(node.Attr)
 		switch strings.ToLower(attribute) {
+		case RegexpOperationMatch:
+			fallthrough
+		case RegexpOperationSearch:
+			return callable(func(args []ast.Expr) (any, error) {
+				if len(args) != 2 {
+					return nil, errors.New("`match` function support exactly two arguments")
+				}
+				c, ok := parsedNode.(callable)
+				if !ok {
+					return nil, errors.New("unsupported node type. has to be callable type")
+				}
+
+				return c(append([]ast.Expr{&ast.Str{
+					S: py.String(strings.ToLower(attribute)),
+				}}, args...))
+			}), nil
```

Comment:
this is not the right place to deal with this — have a look at my comment about making `re` return an `attributeGetter`

Reply:
done.